src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -434,6 +441,22 @@ private boolean isAllowedInSessionLevel(String sectionName, String sessionName,
         }
     }
     
+    private boolean isAllowedInSessionLevelAnySection(String sessionName, String privilegeName) {
+        for (String sectionName : this.sessionLevel.keySet()) {
+            if (this.sessionLevel.get(sectionName).get(sessionName).containsKey(privilegeName) &&
```

Comment:
Should there be a check for <code>this.sessionLevel.get(sectionName).containsKey(sessionName)</code> ?